spec/models/ticket_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:ticket) { create(:ticket, price: 50, conference: conference) }
```

Comment:
Let us explicitely know the currency too. When you test later for same currency in conference, I do not want to have to go search in the factory about which currency this ticket has. Since currency matters for the test you need to let me know. Either here when you create the ticket, or you need to set the currency of the ticket later, before you use ticket variable for the specific test.